### PR TITLE
Feature/random test

### PR DESCRIPTION
**Pull request by Sefiraat**
Another test, you'll note I have spelt Networks wrong...

### CODE REVIEW

**Comment by Sefiraat** [comment]
image

Click this

image

Hover over the line you dont like, and click the +

image

Type in your comment

File: slimefun/addons/networks.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Networks
 
 ![Networks](../../.gitbook/assets/icons\_networks.png)
+
+## About Netwroks
```
Comment (Sefiraat) [inline slimefun/addons/networks.md:5]:
```suggestion
## About Networks
```

**Comment by Sefiraat** [comment]
When you're happy that the whole PR is good, you can then click Start Review in the top left of the changes screen, then click approve and submit. Then you can merge :D 

image

File: slimefun/addons/networks.md (hunk 1 above)
Comment (ReasonFoundDecoy) [inline slimefun/addons/networks.md:5]:
```suggestion
## About Networks
```

**Pull request merged**